{"repository":"https://github.com/chmouel/sugarjazy","dockerfilePath":"Dockerfile","startTime":1672332003558,"clone":{"stdout":"","stderr":"Cloning into '/tmp/dinghy-analysis/chmouel/sugarjazy'...\n","error":null,"commit":"2a2f2cf0a7b580b22fd3bec35869fd430c3a0971"},"originalBuild":{"startTimestamp":1672332005238,"endTimestamp":1672332020905,"error":null,"stdout":"Sending build context to Docker daemon  3.146MB\r\r\nStep 1/6 : FROM python:3.10-alpine\n3.10-alpine: Pulling from library/python\nc158987b0551: Already exists\n7ec3a91b66d9: Already exists\na004f9671752: Pulling fs layer\nf41086029229: Pulling fs layer\na8dbc4976a6c: Pulling fs layer\nf41086029229: Download complete\na8dbc4976a6c: Verifying Checksum\na8dbc4976a6c: Download complete\na004f9671752: Verifying Checksum\na004f9671752: Download complete\na004f9671752: Pull complete\nf41086029229: Pull complete\na8dbc4976a6c: Pull complete\nDigest: sha256:63d0c387f30f82da7e9404b7157b6f077656b5263c8d64c7b7b889f4d8c1c60a\nStatus: Downloaded newer image for python:3.10-alpine\n ---> 2527f31628e7\nStep 2/6 : COPY sugarjazy /code\n ---> c5f2336bd30a\nStep 3/6 : WORKDIR /code\n ---> Running in 5adc0494bd98\nRemoving intermediate container 5adc0494bd98\n ---> 9b04445e86bd\nStep 4/6 : ENV PYTHONFAULTHANDLER=1   PYTHONUNBUFFERED=1   PYTHONHASHSEED=random   PYTHONDONTWRITEBYTECODE=1   PIP_NO_CACHE_DIR=1   PIP_DISABLE_PIP_VERSION_CHECK=1   PIP_DEFAULT_TIMEOUT=100\n ---> Running in 5320fa35defe\nRemoving intermediate container 5320fa35defe\n ---> 226e4d6fe153\nStep 5/6 : RUN pip3 install -U python-dateutil\n ---> Running in 9d84a13e1725\nCollecting python-dateutil\n  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)\n     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 16.0 MB/s eta 0:00:00\nCollecting six>=1.5\n  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)\nInstalling collected packages: six, python-dateutil\nSuccessfully installed python-dateutil-2.8.2 six-1.16.0\n\u001b[91mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\n\u001b[0mRemoving intermediate container 9d84a13e1725\n ---> 9c25aa1347fd\nStep 6/6 : ENTRYPOINT [\"python\", \"cli.py\"]\n ---> Running in 2b9995116575\nRemoving intermediate container 2b9995116575\n ---> 57c2a3e063b0\nSuccessfully built 57c2a3e063b0\nSuccessfully tagged sugarjazy:latest\n","stderr":"","imageSize":50808088},"repair":{"violations":[{"name":"pipUseNoCacheDir","position":{"start":18,"end":18,"columnStart":4,"columnEnd":35}}],"repairedDockerfile":"# This Dockerfile uses multi-stage build to customize DEV and PROD images:\n# https://docs.docker.com/develop/develop-images/multistage-build/\n\nFROM python:3.10-alpine\n\nCOPY sugarjazy /code\nWORKDIR /code\n\nENV PYTHONFAULTHANDLER=1 \\\n  PYTHONUNBUFFERED=1 \\\n  PYTHONHASHSEED=random \\\n  PYTHONDONTWRITEBYTECODE=1 \\\n  # pip:\n  PIP_NO_CACHE_DIR=1 \\\n  PIP_DISABLE_PIP_VERSION_CHECK=1 \\\n  PIP_DEFAULT_TIMEOUT=100\n\n\nRUN pip3 install --no-cache-dir -U python-dateutil\n\nENTRYPOINT [\"python\", \"cli.py\"]\n"},"repairedBuild":{"startTimestamp":1672332021205,"endTimestamp":1672332050634,"error":null,"stdout":"Sending build context to Docker daemon  3.146MB\r\r\nStep 1/6 : FROM python:3.10-alpine\n ---> 2527f31628e7\nStep 2/6 : COPY sugarjazy /code\n ---> 44dc7cafb92b\nStep 3/6 : WORKDIR /code\n ---> Running in 25c41e17677c\nRemoving intermediate container 25c41e17677c\n ---> b921a8fdddce\nStep 4/6 : ENV PYTHONFAULTHANDLER=1   PYTHONUNBUFFERED=1   PYTHONHASHSEED=random   PYTHONDONTWRITEBYTECODE=1   PIP_NO_CACHE_DIR=1   PIP_DISABLE_PIP_VERSION_CHECK=1   PIP_DEFAULT_TIMEOUT=100\n ---> Running in 36ccb0b9ea3e\nRemoving intermediate container 36ccb0b9ea3e\n ---> 21d2e0f31532\nStep 5/6 : RUN pip3 install --no-cache-dir -U python-dateutil\n ---> Running in b3771f3db33b\nCollecting python-dateutil\n  Downloading python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)\n     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 247.7/247.7 kB 20.1 MB/s eta 0:00:00\nCollecting six>=1.5\n  Downloading six-1.16.0-py2.py3-none-any.whl (11 kB)\nInstalling collected packages: six, python-dateutil\nSuccessfully installed python-dateutil-2.8.2 six-1.16.0\n\u001b[91mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\n\u001b[0mRemoving intermediate container b3771f3db33b\n ---> 657f667df24a\nStep 6/6 : ENTRYPOINT [\"python\", \"cli.py\"]\n ---> Running in a0a0b5a5003d\nRemoving intermediate container a0a0b5a5003d\n ---> ff98f72f3afb\nSuccessfully built ff98f72f3afb\nSuccessfully tagged sugarjazy:latest\n","stderr":"","imageSize":50808088},"endTime":1672332050937}